baseline.yaml

```diff
@@ -649,6 +649,29 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+  - id: OSPS-DO-13
+    maturity_level: 1
+    category: Documentation
+    criteria: |
+       The PROJECT DOCUMENTATION MUST include a 
+       descriptive statement about how the project 
+       is supported/maintained and what is not supported. 
+       Projects are encouraged to provide "End of 
+       Support" statements when versions or features 
+       are deprecated or no longer supported.
```

Comment:
Encouragements aren't controls. This needs rephrased or moved to the objective / implementation.

Reply:
I'd LIKE to make it a requirement, but understand that not all projects send out EOL notices.  Should this be adjust to be a more directive statement or split out to a higher level criteria by itself?

Reply:
I'd support sending EOL notices as a level 3 (arguably 2, but I'd like to start with 3 and be talked into 2) requirement.

Reply:
OK, how about we split out that part to lvl and leave the broader "provide support statement" for lvl1?

Reply:
Works for me.

Reply:
adjusted and will propose additional criteria for EoS statement